CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+
+These are the values we embed at every level of our stack:
+
+### Seven-generation thinking
+We are building tools for long-term planetary thriving; so that our children have clean air to breath and healthy food to eat.
+
+### Relationship-centered design
+We work in deep partnership with all stakeholders, optimizing for trust and relationship-building with every other community this work touches. We do this by prioritizing co-creative product and design sessions, sharing a transparent product journey, and inviting suggestions and engagement from all Hylo stakeholders to ensure we are building for real needs of real communities.
+
+### Transparency
+Hylo is open source and our operations, finances, and governance are open to all.
+
+### Interoperability
+We contribute to and create open standards and protocols wherever possible, and work to integrate with as many existing platforms as we can, especially those which are open source and values aligned.
+
+### Privacy and User Sovereignty
+Your data belongs to you, not us. We will never sell your data or share it without your explicit request. You can export and delete your data as you wish.
+
+### Inclusivity
+We are committed to ensuring these tools are shaped by and effectively serve diverse communities. Hylo will be accessible to all by remaining free to use in perpetuity. To support the development, we lean into the Gift and ask every stakeholder to contribute what they can.
+
+### Collective Stewardship
+Hylo is already developed by consensus among the Terran Collective stewards, and in deep partnership with our users. Eventually, Hylo will be stewarded by the community as a platform cooperative, where all users are stakeholders who may participate in the collective governance of the platform and receive abundance from its success.
+
+If these resonate with you, we’d love for you to join the community of Hylo contributors.
+
+## How to help
+There are so many ways to contribute to Hylo, and we’d be delighted for you to participate in whatever way is most exciting to you! Of course, as a software project, we are always welcoming volunteer coders. But you don’t have to contribute code to be helpful! Regardless of your technical chops, there’s plenty to do.
+
+### Here are some important ways to help out:
+- Beta test new releases to ensure high quality functionality
+- Create documentation, such as onboarding materials to help communities thrive on Hylo
+- Submit feedback, bug reports, and feature requests
+- Contribute code!
+
+To get involved in beta testing, creating onboarding materials, or offering feedback on product ideas or designs, join [Building Hylo](https://www.hylo.com/groups/building-hylo/join/RcDe6vcG7u).
```

Comment:
a little nervous about making this invite public... but ok

Reply:
what's your alternative suggestion for people to get involved?

Reply:
nothing right now